client/src/components/Connections/Connections.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link } from 'react-router-dom'
+import "./Connections.css";
+import ProfilePicture from '../ProfilePicture/ProfilePicture'
+
+// function Based React Component
+function Connections(props) {
+  let heading = "Connections"
+
+
+  if (props.userViewing === true) {
+    heading = "My Connections"
+  } else if (props.userViewing === false) {
+    heading = "Her Connections"
+  } else if (!props.userViewing) {
+    heading = "Members"
+  }
+
+  let userArray = props.users.map((person, index) => {
+    let userLastInitial = person.userSurname.slice(0, 1) + '.'
+    if (index < 4) {
+      return (
+        <div key={person.userId} className='Connections__person'>
+          <ProfilePicture image={person.userPic} />
+          <p className='Connections--name'>{person.userFirstName} {userLastInitial}</p>
+        </div>
+      )
+    } else if (index === 4) {
+      return (
+        <div className='Connections__person'>
+          <Link className='Connections__link' to={props.to}>
+            {props.extraUsers}
```

Comment:
Center this text vertically. It is appearing at the top of the div at the moment.